stdlib/benchmarks/collections/bench_string.mojo

```diff
@@ -16,8 +16,9 @@ from benchmark import Bench, BenchConfig, Bencher, BenchId, Unit, keep, run
 from random import random_si64, seed
 from pathlib import _dir_of_current_file
 from collections import Optional
-from utils._utf8_validation import _is_valid_utf8
 from os import abort
+from stdlib.collections.string import String
```

Comment:
I've reverted the explicit `stdlib.` in the import internally - it's not needed.  When you say `from collections.string import String` for example, the compiler implicitly looks within the `stdlib` mojopkg first in the compiler's name lookup.  In other words, all imports implicitly check stdlib first, priority wise essentially.